BiasAndFairnessServers/src/controllers/bias_and_fairness.py

```
@@ -676,14 +711,109 @@ async def get_bias_fairness_evaluation_by_id_controller(eval_id: str, tenant: st
+                )
+            
+            # Use the correct schema name
+            schema_name = "a4ayc80OGd" if tenant == "default" else tenant
+            
+            result = await db.execute(
+                text(f'''
+                    UPDATE "{schema_name}".bias_fairness_evaluations 
+                    SET config_data = :config_data, updated_at = CURRENT_TIMESTAMP
+                    WHERE eval_id = :eval_id
+                    RETURNING id
+                '''),
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)